src/main/java/org/candlepin/subscriptions/tally/CloudigradeAccountUsageCollector.java

```diff
@@ -103,7 +103,7 @@ private void enrichUsageWithCloudigradeData(Map<String, AccountUsageCalculation>
                     calculation.addCloudigrade(HardwareMeasurementType.AWS_CLOUDIGRADE, count);
                 }
                 catch (IllegalArgumentException e) {
-                    log.warn("Skipping cloudigrade usage: {}", usageCount, e);
+                    log.warn("Skipping cloudigrade usage: {}, due to: {}", usageCount, e.toString());
```

Comment:
I'm Ok with this, but IIRC there was concern about the log messages cluttering the log. I'd suggest perhaps logging the 'usageCount' string as debug.